### ISSUE TITLE
Taxonomy custom meta fields returning false from REST API

### ISSUE THREAD

**Issue by moin5050** [pods-framework/pods#3365]
Hi,
I am using Pods (V 2.6) and WP Rest API (V 2.0-beta11).
I have extended a taxonomy called **client** and i add custom fields to it (granted "Read via REST API" access) . 
The fields are **client_email** & **client_address** . the problem is even after having a value stored in this fields the response is **false**.

Kindly help me out. Thanks in advance

![api-error](https://cloud.githubusercontent.com/assets/8633087/12618107/7eb8f0d4-c538-11e5-9681-939f55827f29.png)

**Comment by jimtrue** [comment]
I can confirm as I just tested this, Custom Post Types are returning values, but custom meta added to a Custom Taxonomy is not.

**Comment by anandamd** [comment]
The issue lies in `classes/PodsRESTHandlers.php` on line #58 . That line of code "expects" the object to be a CPT by looking for the `type` key. Add the following code after line #58
```

		$pod_name = pods_v( 'type', $object ); // LINE #58

                // if $pod_name is empty then the route invoked is for a taxonomy, so act accordingly.
		if( empty( $pod_name ) ) {
			$pod_name = pods_v( 'taxonomy', $object );
		}
```

**Comment by jimtrue** [comment]
Corrected by #3368
